prisma/schema.prisma

```diff
@@ -9,12 +9,6 @@ datasource db {
   schemas  = ["download", "public"]
 }
 
-model alembic_version {
-  version_num String @id(map: "alembic_version_pkc") @db.VarChar(32)
-
-  @@schema("public")
-}
-
```

Comment:
This will need to exist since we use alembic for API migrations

Reply:
This model should be back now